lib/locales/en.ts

```diff
@@ -507,6 +519,8 @@ export const locale: Locale = {
     innocent_v_pope: 'Blessed Innocent V, Pope',
     innocent_xi_pope: 'Blessed Innocent XI, Pope',
     irenaeus_of_lyon_bishop: 'Saint Irenaeus, Bishop, Martyr and Doctor of the Church',
+    irenaeus_of_lyon_bishop_patron_of_diocese_of_lyon_and_companions_martyrs:
+      'Saints Ireneus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese, and Companions, Martyrs',
```

Comment:
1. There is always a definite article (`the`) before a name of a diocese.
2. Use the full name of the diocese (both in key and value; in `fr.ts` you should use the celebration name as it is used in the official liturgical books).
3. In English, `Irenaeus` spelling is prefered over `Ireneus`.

```suggestion
    irenaeus_of_lyon_bishop_patron_of_the_archdiocese_of_lyon_and_companions_martyrs:
      'Saints Irenaeus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```

Reply:
I've added the diocese name both in `en` and `fr`.